modules/structs/hook.go

```diff
@@ -19,6 +19,7 @@ var ErrInvalidReceiveHook = errors.New("Invalid JSON payload received over webho
 type Hook struct {
 	ID                  int64             `json:"id"`
 	Type                string            `json:"type"`
+	BranchFilter        string            `json:"branch_filter" binding:"GlobPattern"`
```

Comment:
What is `binding:"GlobPattern"`?

Reply:
I don't know, I just copied from https://github.com/go-gitea/gitea/blob/79c3329502217b664ef81d3f7f65aaf58282d29d/modules/structs/hook.go#L48